sw/device/silicon_creator/rom/e2e/presigned_images/BUILD

```diff
@@ -13,3 +13,53 @@ filegroup(
     name = "presigned_fpga_cw310_bin",
     srcs = ["empty_test_sigverify_spx_fpga_cw310.fake_rsa_prod_key_0.fake_spx_prod_key_0.signed.bin"],
 )
+
+filegroup(
+    name = "empty_test_fpga_cw310",
+    srcs = ["empty_test_slot_a_fpga_cw310.signed.bin"],
+)
+
+filegroup(
+    name = "empty_test_sim_dv",
+    srcs = ["empty_test_slot_a_sim_dv.signed.bin"],
+)
+
+filegroup(
+    name = "empty_test_sim_verilator",
+    srcs = ["empty_test_slot_a_sim_verilator.signed.bin"],
+)
+
+filegroup(
+    name = "rom_e2e_self_hash_fpga_cw310",
+    srcs = ["rom_e2e_self_hash_fpga_cw310.signed.bin"],
+)
+
+filegroup(
+    name = "rom_e2e_self_hash_sim_dv",
+    srcs = ["rom_e2e_self_hash_sim_dv.signed.bin"],
+)
+
+alias(
```

Comment:
just dropping a note here: this alias is only needed b/c the `opentitan_functest` rule expects to find a target with the name `*_<device>_bin` when `signed = False` and `*_<device>_signed_bin` when `signed = True`, but if the above filegroups were named accordingly, then the aliases would not be needed I don't think (we should probably drop a comment explaining this so these label names are not updated in the future.

Reply:
yeah, good point. That's why I added those when I first wrote this file but forgot to add a comment.

Reply:
@cfrantz tried this and it did not work, we will investigate in a follow up.